### DISCUSSION TITLE
Datasheet Download/Open

### DISCUSSION THREAD

**Discussion by kangaroosinaustria** [inventree/InvenTree#7897] [Ideas]
It's entirely possibly I haven't looked hard enough, but at the moment it seems that the only option for viewing a datasheet that has been attached within Inventree, is to download the file.
Obviously for some attachment types, download makes sense as if you've associated a hex file with an item, there's no point trying to open that in the browser.
However I often find it much more convenient to simply view a datasheet within the browser rather than downloading it each time.

 I tried changing the live HTML within the browser but still didn't have any luck. I'm not a developer so short of diving into the django documentation, is there perhaps an environment variable that could be set to change the default behaviour or allow for choosing open/download each time?
Would it be feasible to perhaps make this available as an environment variable in the future?

**Comment by matmair** [comment]
Duplicate of https://github.com/inventree/InvenTree/issues/7894

**Reply by kangaroosinaustria** [reply]
Thanks, I don't know why I couldn't find that when I searched. Makes sense
